tests/comments_closure_typecast/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -125,11 +140,11 @@ const data = functionCall(
 );
 
 // Invalid type casts
-const v1 = /** @type {} */ value;
-const v2 = /** @type {}} */ value;
-const v3 = /** @type } */ value;
-const v4 = /** @type { */ value;
-const v5 = /** @type {{} */ value;
+const v1 = /** @type {} */ (value);
+const v2 = /** @type {}} */ (value);
+const v3 = /** @type } */ (value);
+const v4 = /** @type { */ (value);
+const v5 = /** @type {{} */ (value);
```

Comment:
We should ignore it and keep as is

Reply:
I don't think we should. We might try checking the syntax of types in these simple cases, but what about more complex ones? Those types can contain everything, e.g. curly braces inside quotes.

Reply:
That's what I mean:
```js
const curly = /** @type { '}' | '{' | '}}' } */ (getCurly());
```

Reply:
Also, as I wrote in comments in the code:
> TypeScript expects the type to be enclosed in curly brackets,
> however Closure Compiler accepts types in parens and even without any delimiters at all.
> That's why we only check that there is at least one non-whitespace character after "@type".

Reply:
Sound reasonable, but on the other hand we can add unnecessary parens when somebody has `@type` in comments, yes, it is edge case, let's keep it as is, if somebody open a issue about it we will improve it

Reply:
This is a very edge case. One has to have both `@type` in a comment AND a parenthesized expression after that comment.